components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/latest/GitpodTerminalService.kt

```diff
@@ -164,7 +153,7 @@ class GitpodTerminalService(session: ClientProjectSession) : Disposable {
 
             val listTerminalsResponseFuture = terminalServiceFutureStub.list(listTerminalsRequest)
 
-            lifetime.onTerminationOrNow {
+            session.lifetime.onTerminationOrNow {
```

Comment:
we should not cancel it anymore if we don't care about clients? or only if project unloaded?